libexec/klab-build

```diff
@@ -46,7 +47,9 @@ const config_json   = JSON.parse(fs.readFileSync("./config.json"));
 const config        = makeConfig(config_json);
 const config_path   = cmd["<spec>"] || config.src.specification;
 const filter_subject= cmd["--filter"] || null;
+const OOG           = cmd["--oog"] || false;
```

Comment:
default behaviour should be to exhaust the search space; generating the oog spec should be default